### PR TITLE
Lean Helix flakiness upgrade

### PR DESCRIPTION
**Pull request by ronnno**
* points to the latest orbs-network/lean-helix-go#86
* upgrades membuffers to v0.3.4

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -22,10 +22,11 @@ require (
 	github.com/karalabe/hid v1.0.0 // indirect
 	github.com/mattn/go-colorable v0.1.4 // indirect
 	github.com/mattn/go-isatty v0.0.10 // indirect
+	github.com/orbs-network/contract-external-libraries-go v0.0.0
```
Comment (electricmonk) [inline go.mod]: why is this added to go mod?

Comment (ronnno) [reply go.mod]: removed

File: test/acceptance/lean_helix_recovers_from_disk_error_test.go [hunk 2]
```diff
@@ -25,6 +25,9 @@ func TestLeanHelix_RecoversFromDiskWriteError(t *testing.T) {
 		AllowingErrors(
 			"failed to commit block received via sync",
 			"cannot get elected validators from system contract", // LH tries to read state from a block height that has not been properly persisted and therefore, fails
+			".*contract returned error.*",
+			".*cannot get ordered committee.*",
+			".*",
```
Comment (vistra) [inline test/acceptance/lean_helix_recovers_from_disk_error_test.go]: accidentaly committed, fix and remove
